examples/nvme/perf/perf.c

```diff
@@ -899,6 +899,19 @@ register_workers(void)
 	return 0;
 }
 
+static void
+unregister_worker(void)
```

Comment:
Please use "workers" rather than "worker" for consistency with register_workers()